test/e2e/play_kube_test.go

```diff
@@ -4984,6 +4984,18 @@ spec:
 		exec.WaitWithDefaultTimeout()
 		Expect(exec).Should(Exit(0))
 		Expect(exec.OutputToString()).Should(Equal("hi"))
+
+		teardown := podmanTest.Podman([]string{"kube", "down", "--force", kubeYaml})
+		teardown.WaitWithDefaultTimeout()
+		Expect(teardown).Should(Exit(0))
```

Comment:
Can we also make sure that removed volumes are mentioned in the output of `kube down --force`?